statistics/cmsketch.go

```diff
@@ -688,6 +688,18 @@ func (c *TopN) RemoveVal(val []byte) {
 	c.TopN = append(c.TopN[:pos], c.TopN[pos+1:]...)
 }
 
+// MemoryUsage returns the total memory usage of a topn.
+func (c *TopN) MemoryUsage() (sum int64) {
+	if c == nil {
+		return
+	}
+	sum = 32
```

Comment:
Why 32 here？ `unsafe.Sizeof(TopN{})` seems to be 24.

Reply:
include reference...

Reply:
add comments